# 
# @brief    : VISION_POSITION
# @time     : 2018/09/26
# @auhtor   : liuxu <liuxu.ccc@gmail.com> 
#

# position gotten from vision

float32 x          #point X position, in m
float32 y          #point Y position, in m
float32 z          #point Z position, in m

